x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/SpatialPlugin.java

```diff
@@ -172,7 +176,15 @@ public List<AggregationSpec> getAggregations() {
                     SpatialStatsAction.Item.CARTESIANCENTROID,
                     checkLicense(CartesianCentroidAggregationBuilder.PARSER, GEO_CENTROID_AGG_FEATURE)
                 )
-            ).addResultReader(InternalCartesianCentroid::new).setAggregatorRegistrar(this::registerCartesianCentroidAggregator)
+            ).addResultReader(InternalCartesianCentroid::new).setAggregatorRegistrar(this::registerCartesianCentroidAggregator),
+            new AggregationSpec(
+                CartesianBoundsAggregationBuilder.NAME,
+                CartesianBoundsAggregationBuilder::new,
+                usage.track(
+                    SpatialStatsAction.Item.CARTESIANBOUNDS,
+                    CartesianBoundsAggregationBuilder.PARSER    // TODO: Should we have a license for this?
+                )
```

Comment:
No, license for CartesianBounds should be the same as for GeoBounds, so it is basic.